static/docs/commands-reference/repro.md

```diff
@@ -166,7 +166,7 @@ $ tree
 ├── count.txt      <---- result: "2"
 ├── filter.dvc     <---- first stage
 ├── numbers.txt    <---- intermediate result of the first stage
-├── process.py     <---- code that runs some transformation
+├── process.py     <---- code that causes data transformation
```

Comment:
causes is not the best term either. If we want to change this we can use `implements` for example or even simple `does`.

Reply:
I see you've fixed this already (:  Thanks